### DISCUSSION TITLE
timeAgo function has error

### DISCUSSION THREAD

**Discussion by behzad-ahmadi** [persian-tools/persian-tools#319] [General]
timeAgo fucntion has error.

I use like this:   

`const t = timeAgo("1348/10/30 14:10:27")
`
the error : TypeError: PersianTools: convertToTimeStamp - The input format must be yyyy/mm/dd hh:mm:ss

**Comment by ali-master** [comment]
Thanks for reporting, we will check it and come back to you

**Reply by behzad-ahmadi** [reply]
Welcome, I'm waiting for you. I really need this function for my project.
